Binary files swig_original/.git/index and swig/.git/index differ
diff -uNr swig_original/.git/ORIG_HEAD swig/.git/ORIG_HEAD
--- swig_original/.git/ORIG_HEAD	2016-05-15 18:24:45.679568751 +0800
+++ swig/.git/ORIG_HEAD	1970-01-01 08:00:00.000000000 +0800
@@ -1 +0,0 @@
-809d54b13a1e032857838efbcbf6162b64c63984
diff -uNr swig_original/Source/Modules/go.cxx swig/Source/Modules/go.cxx
--- swig_original/Source/Modules/go.cxx	2016-05-15 18:24:45.666235417 +0800
+++ swig/Source/Modules/go.cxx	2016-05-15 18:11:11.339536341 +0800
@@ -6525,8 +6525,10 @@
 	break;
       }
     }
-
-    if (Getattr(undefined_types, ty) && !Getattr(defined_types, ty)) {
+    
+    String* go_type = goType(n, ty);
+    
+    if (Getattr(undefined_types, go_type) && !Getattr(defined_types, go_type)) {
       return goWrapperType(n, type, true);
     }
 
